.github/workflows/push.yml

```diff
@@ -121,7 +121,7 @@ jobs:
           configurationParameters: >-
             --${{ matrix.debug && 'enable' || 'disable' }}-debug
             --${{ matrix.zts && 'enable' || 'disable' }}-zts
-            ${{ matrix.asan && 'CFLAGS="-fsanitize=undefined,address -fno-sanitize=pointer-overflow -DZEND_TRACK_ARENA_ALLOC" LDFLAGS="-fsanitize=undefined,address -fno-sanitize=pointer-overflow" CC=clang-16 CXX=clang++-16' || '' }}
+            ${{ matrix.asan && 'CFLAGS="-Wconditional-uninitialized -fsanitize=undefined,address -fno-sanitize=pointer-overflow -DZEND_TRACK_ARENA_ALLOC" LDFLAGS="-Wconditional-uninitialized -fsanitize=undefined,address -fno-sanitize=pointer-overflow" CC=clang-16 CXX=clang++-16' || '' }}
```

Comment:
I don't think adding this to `LDFLAGS` is needed. `fsanitize` is only added because Clang runs into linker issues without it. I'll have a look why that is.

Reply:
Ah OK, thanks. TBH I didn't really know what the difference is between the two flags :/...